dle/data/migrations/0006_alter_productsection_bert_vector.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.0.2 on 2023-04-01 05:25
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('data', '0005_productsection_bert_vector'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='productsection',
```

Comment:
It looks like the same migration is in here twice - the only difference with the second one having `blank=True`

Reply:
Yep, it needs to be `blankable`. We could try to [squash the migrations](https://docs.djangoproject.com/en/4.1/topics/migrations/#migration-squashing) but not sure if it's currently worth it